nav2_msgs/action/FollowPath.action

```diff
@@ -1,10 +1,21 @@
+# Error codes
+# Note: The expected priority order of the errors should match the message order
+int16 NONE=0
```

Comment:
For ordering,

None, Unknown, TF, invalid path, patience, progress, no valid control?